msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -70,9 +70,9 @@ For more details, see [chat resource-specific consent permissions](/graph/permis
 
 ## Enable RSC in your application
 
-1. [Configure consent settings in the Azure AD portal](#configure-consent-settings-in-the-azure-ad-portal).
-    1. [Configure group owner consent settings for RSC in a team](#configure-group-owner-consent-settings-for-rsc-in-a-team).
-    1. [Configure user consent settings for RSC in a chat](#configure-user-consent-settings-for-rsc-in-a-chat).
+1. [Configure consent settings](#configure-consent-settings).
+    1. [Configure group owner consent settings for RSC in a team (via Azure AD portal)](#configure-group-owner-consent-settings-for-RSC-in-a-team-via-Azure-AD-portal).
+    1. [Configure chat owner consent settings for RSC in a chat (via Microsoft Graph APIs)](#configure-chat-owner-consent-settings-for-RSC-in-a-chat-via-Microsoft-Graph-APIs).
```

Comment:
configure-chat-owner-consent-settings-for-rsc-in-a-chat-via-microsoft-graph-apis